src/workflows/recipes/components/DetailsActionBar.js

```diff
@@ -22,15 +25,15 @@ import { reverse } from 'console/urls';
   (state, props) => {
     const recipeId = getUrlParamAsInt(state, 'recipeId');
     const latestRevisionId = getLatestRevisionIdForRecipe(state, recipeId, '');
-    const recipe = getRecipe(state, recipeId, new Map());
+    const currentRevision = getCurrentRevisionForRecipe(state, recipeId, new Map());
```

Comment:
I'm not sure I understand the question here. My guess is you're wondering why the default value of `new Map()`? This is just an edge case fallback in case the recipe hasn't fully loaded. A recipe would never not have at least one revision so `latest_revision` would never be `null`. But maybe that's not what you're asking about here?

Reply:
That *is* what I'm asking about. I'm also suggesting we remove that fallback to `new Map()` since it will never trigger and if it ever does, we most likely have another problem somewhere else. 
Perhaps that's a bit anal. My underlying concern is that a code-reader might see this and wonder "Why the heck would a recipe *not* have a revision?! What was the reasoning behind that."

Totally up to you. Either add a code-comment, delete it, or ignore my overly sensitive concerns :)